.github/workflows/ci.yml

```diff
@@ -44,6 +44,24 @@ jobs:
           echo "Changed files:"
           cat changed_files.txt
 
+          # Run all jobs if CI configuration changes
+          if grep -q '^\.github/' changed_files.txt; then
+            echo "jobs_to_run=$jobs" >> $GITHUB_OUTPUT
+            exit 0;
+          fi
+
+          # Run all jobs if tool versions change
+          if grep -q '^\.tool-versions' changed_files.txt; then
+            echo "jobs_to_run=$jobs" >> $GITHUB_OUTPUT
+            exit 0;
+          fi
+
+          # Run all jobs on branches that start with "bust"
```

Comment:
I am not convinced about this. You can just manually trigger the workflow for your branch to achieve the same thing?

Reply:
👍  added manual triggers to elixir and rust